docs/source/howto/index.rst

```diff
@@ -9,3 +9,4 @@ How-to guides help you to solve specific problems with pdfminer.six.
     :maxdepth: 1
 
     images
+    aforms
```

Comment:
Just to keep things nice and explicit, I'd rename the file to `acro_forms`.

Reply:
Will do